.github/CONTRIBUTING.md

```diff
@@ -1,3 +1,24 @@
+# Contributions
+
+We welcome contributions in the form of issues and pull requests.  We view the contributions and the process as the same for github and external contributors.
+
+## Issues
+
+Log issues for both bugs and enhancement requests.  Logging issues are important for the open community.
+
+Issues in this repository should be for the toolkit packages.  Runner specific issues can be filed [in the runner repository](https://github.com/actions/runner).
+
+## Enhancements and Feature Requests
+
+We ask that before significant effort is put into code changes, that we have agreement on taking the change before time is invested in code changes. 
+
+1. Create a feature request.  Once agreed we will take the enhancment
+2. Create an ADR to agree on the details of the change.
+
+An ADR is an Architectural Decision Record.  This allows consensus on the direction forward and also serves as a record of the change and motivation. [Read more here](docs/adrs/README.md).
```

Comment:
Does this relative link work? Weird because under .github/ folder but i guess depends on whatever url when issue template is loaded?